.github/workflows/main.yaml

```diff
@@ -67,3 +73,6 @@ jobs:
 
       - name: Test
         run: make test
+      
+      - name: Search index dry run
```

Comment:
We want to run this as a pr_check. This script should be added to the `Dockerfile` after the `make validate-schema` to block the Jenkins build and failing the checks/build.

Reply:
Alright. My logic here was that the PR check would be a bit more concise as to why the PR is failing (versus having to dive into Jenkins and interpret the output). 

For now I stuck it in the Dockerfile. If it looks good to you I'll fix the various issues so this PR passes.

Reply:
Yup sounds good.